cdc/puller/multiplexing_puller.go

```diff
@@ -71,6 +73,14 @@ type tableProgressWithSubID struct {
 	subID kv.SubscriptionID
 }
 
+// Puller pull data from tikv and push changes into a buffer.
```

Comment:
Seems it can be removed.

Reply:
Thank you, I will remove it in next code improvement pull request.